controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -24,16 +24,22 @@ def recording_path() -> Path:
     date = now.date().isoformat()
     # Windows doesn't like colons in filenames
     name = now.isoformat().replace(':', '-')
-    return Path(date) / name / '{}.html'.format(name)
+    return Path(date) / name / '{}'.format(name)
 
 
 @contextlib.contextmanager
 def record_animation(supervisor: Supervisor, file_path: Path) -> Iterator[None]:
```

Comment:
We should probably also rename this.

Or (and from an API/clarity perspective this may be preferable) we could have one context manager for the animation and another for the video. That could be preferable in case we need to turn one of them off for some reason -- it'd just be removing a single context manager to remove one, rather than needing to work out which lines in here should be changed.